changelog.d/13597.misc

```diff
@@ -0,0 +1 @@
+Optimise push action fetching queries. Contributed by Nick @ Beeper (@fizzadar).
```

Comment:
Would `get_unread_push_actions_for_user_in_range_for_email` benefit from these changes too? I think the queries are identical except for columns returned.

Reply:
Yes! Not sure how I missed that... https://github.com/matrix-org/synapse/pull/13597/commits/34be471443c70a6111d0c772e9413bc3a9ef01fe